packages/core/useKeyModifierState/index.ts

```diff
@@ -1,24 +1,35 @@
 import { ref } from 'vue-demi'
 import { useEventListener, WindowEventName } from '../useEventListener'
+import { ConfigurableDocument, defaultDocument } from '../_configurable'
 
 export type Modifier = 'Alt' | 'AltGraph' | 'CapsLock' | 'Control' | 'Fn' | 'FnLock' | 'Meta' | 'NumLock' | 'ScrollLock' | 'Shift' | 'Symbol' | 'SymbolLock'
 
 const defaultEvents: WindowEventName[] = ['mousedown', 'mouseup', 'keydown', 'keyup']
 
-export interface ModifierOptions {
+export interface ModifierOptions extends ConfigurableDocument {
   /**
    * Event names that will prompt update to modifier states
    *
    * @default ['mousedown', 'mouseup', 'keydown', 'keyup']
    */
   events?: WindowEventName[]
+
+  /**
+   * Initial value of the returned ref
+   *
+   * @default null
+   */
+  initial?: null | boolean
+
 }
 
 export function useKeyModifierState(modifier: Modifier, options: ModifierOptions = {}) {
   const {
     events = defaultEvents,
+    document = defaultDocument,
+    initial = null,
   } = options
-  const state = ref<null | boolean>(null)
+  const state = ref(initial)
```

Comment:
I suppose this will make the ref lose the type of `boolean`, no?

Reply:
[e799825](https://github.com/vueuse/vueuse/pull/698/commits/e79982510090845f78149310ce026eb29451c61a)